src/footer/Footer.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import './Footer.less';
+
+export default class Footer extends React.Component<any, any> {
```

Comment:
footer component is a simple component without any logic, use function components instead of classes